views/teams/topics.hbs

```diff
@@ -128,7 +128,7 @@
             {{/userHasPermission}}
             <h2 class="h4">Tools</h2>
 
-            <div class="list-group">
+            <div class="list-group" data-testid="tool_list">
```

Comment:
see above for detailed name for "tool_list"

Reply:
changed